templates/change_role.html

```diff
@@ -40,12 +40,12 @@ <h5 class="greeting" style="box-sizing: border-box;margin-top: 0;margin-bottom:
                 </div>
                 <div class="rectangle-3" style="box-sizing: border-box;margin: 60px 138px;">
                     <p class="invite" style="box-sizing: border-box;margin-top: 0;margin-bottom: 1rem;orphans: 3;widows: 3;color: #6d6d6d;line-height: 21px;font-family: 'DINPro', 'Roboto', sans-serif !important;">
-                        You have been added as an Admin to Converge
+                        Your role on Converge has been changed to {{new_role}}.
                         <br><br>Regards,<br><strong>Converge Team</strong>
                     </p>
                 </div>
                 <div style="box-sizing: border-box;">
-                    <a href=http://converge-staging.andela.com/> <button type=" button" class="btn btn-primary" style="box-sizing: border-box;border-radius: 4px;margin: 0;font-family: 'DINPro', 'Roboto', sans-serif;font-size: 1rem;overflow: visible;text-transform: none;-webkit-appearance: button;display: inline-block;font-weight: 400;text-align: center;white-space: nowrap;vertical-align: middle;-webkit-user-select: none;-moz-user-select: none;-ms-user-select: none;user-select: none;border: 1px solid transparent;padding: .375rem .75rem;transition: color .15s ease-in-out,background-color .15s ease-in-out,border-color .15s ease-in-out,box-shadow .15s ease-in-out;color: #fff;background-color: #007bff;border-color: #007bff;height: 44px;width: 174px;">Login
+                    <a href={{domain}}> <button type=" button" class="btn btn-primary" style="box-sizing: border-box;border-radius: 4px;margin: 0;font-family: 'DINPro', 'Roboto', sans-serif;font-size: 1rem;overflow: visible;text-transform: none;-webkit-appearance: button;display: inline-block;font-weight: 400;text-align: center;white-space: nowrap;vertical-align: middle;-webkit-user-select: none;-moz-user-select: none;-ms-user-select: none;user-select: none;border: 1px solid transparent;padding: .375rem .75rem;transition: color .15s ease-in-out,background-color .15s ease-in-out,border-color .15s ease-in-out,box-shadow .15s ease-in-out;color: #fff;background-color: #007bff;border-color: #007bff;height: 44px;width: 174px;">Login
```

Comment:
@mnswaleh The `Login to view` button is not clickable. You can look into this.

Reply:
Did you configure `DOMAIN_NAME` on your `.env` @rajeman

Reply:
Sure I did. Try hitting the mutation within docker to confirm.

Reply:
let me check into that then

Reply:
@mnswaleh This a great job. The query that changes the user's role works perfectly, however, the one that invites them to join converge return a successful response but does not send an email. Please help me understand if I have the wrong set up or if there is any problem.